txircd/modules/rfc/cmd_away.py

```diff
@@ -16,6 +17,20 @@ def actions(self):
 		return [ ("commandextra-PRIVMSG", 10, self.notifyAway),
 				("commandextra-NOTICE", 10, self.notifyAway),
 				("extrawhois", 10, self.addWhois) ]
+
+	def load(self):
+		self.rehash()
+
+	def rehash(self):
+		self.ircd.isupport_tokens["AWAYLEN"] = self.ircd.config.get("away_length", 200)
+
+	def verifyConfig(self, config):
+		if "away_length" in config:
+			if not isinstance(config["away_length"], int) or config["away_length"] < 0:
+				raise ConfigValidationError("away_length", "invalid number")
+			elif config["away_length"] > 200:
+				config["away_length"] = 200
```

Comment:
This won't work on load since `__setitem__` isn't defined for Config. (This will just raise an error.)